modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/ProjectsList/components/ProjectCard/ProjectCard.js

```diff
@@ -27,112 +27,139 @@ const statusReport = {
 	[SLA_STATUS_TYPES.expired]: i18n.translate('ended-on'),
 };
 
-const ProjectCard = ({compressed, loading, onClick, ...koroneikiAccount}) => (
-	<ClayCard
-		className={classNames(
-			'border border-brand-primary-lighten-4 card-interactive shadow-none',
-			{
-				'card-horizontal mb-3': compressed,
-				'cp-project-card-lg mr-5 mb-4': !compressed,
-			}
-		)}
-		onClick={onClick}
-	>
-		<ClayCard.Body
-			className={classNames({
-				'mx-2 py-4 my-3': !compressed,
-				'py-4': compressed,
-			})}
+const ProjectCard = ({compressed, loading, onClick, ...koroneikiAccount}) => {
+	const getSLAStatus = () => {
+		if (loading) {
+			return <Skeleton height={20} width={54} />;
+		}
+
+		if (
+			!koroneikiAccount.hasSLAGoldPlatinum ||
+			koroneikiAccount.status === 'Expired'
+		) {
+			return '';
+		}
```

Comment:
This code is repeat, could be a variable

```suggestion
		if (
			!koroneikiAccount.hasSLAGoldPlatinum ||
			koroneikiAccount.status === 'Expired'
		) {
			return '';
		}
```